src/Events/REST/TEC/V1/Endpoints/Event.php

```diff
@@ -68,9 +68,6 @@ class Event extends Post_Entity_Endpoint implements RUD_Endpoint {
 	 * @return WP_REST_Response The response object.
 	 */
 	public function update( array $params = [] ): WP_REST_Response {
-		// Transform venues array to use first element (TEC default behavior).
```

Comment:
Everything that is here already happens in the update method of the trait. So it makes the overwriting here not adding any value.